apps/condo/domains/meter/gql.js

```diff
@@ -52,6 +52,12 @@ const DELETE_METER_AND_METER_READINGS_MUTATION = gql`
 const METER_RESOURCE_OWNER_FIELDS = `{ organization { id } resource { id } address addressKey ${COMMON_FIELDS} }`
 const MeterResourceOwner = generateGqlQueries('MeterResourceOwner', METER_RESOURCE_OWNER_FIELDS)
 
+const ALL_RESIDENT_METERS_QUERY = gql`
+    query allResidentMeters ($where: ResidentWhereUniqueInput!, $first: Int, $skip: Int) {
```

Comment:
Maybe call first argument resident for all mobile users? and N optional args for query-specific logic